requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,70 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a more impactful launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+- Notebooks side panel
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+
+## Rollout plan
+- [ ] Base 3000 ready to release (10th Nov)
+- [ ] Internal and a/b testing starts (13th Nov)
+- [ ] Docs revamp and updates ready (24th Nov)
+- [ ] Product pages for 3000 / Notebooks (24th Nov)
+- [ ] Release [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) (27th Nov)
+- [ ] Release [Notebooks](https://app.posthog.com/feature_flags/8734) (4th Dec) (or together with Base 3000 if we feel docs are easier)
+
+
+## Pre-launch evaluation
+
+We decided that even the "base" version has enough major changes that we want to test that it both doesn't impact our key metrics negatively as well as (if possible) evaluate if it is overall an improvement, especially with new users.
+This might be hard to do or to get clear signals for whether it is an improvement (especially as it is only a step along the way to the end goal UI/UX).
+
+- [ ] **Signup/onboarding Experiment**: _just_ for the onboarding we could experiment with the styles separate from the rest of the app
+  - [ ] **Metrics**: Same as usual onboarding metrics?
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We mostly care about **negatively** impacting the flow
+- [ ] **General Experiment**: in combination with internal testing we could start to roll out the product to new and existing users as an experiment
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We originally opted against being able to go back to the old UI - do we want to offer this?
+  - [ ] **Metrics**:
+    - [ ] Same as onboarding?
+    - [ ] Retention over time (what time frame do we need to be clear on this?)
+    - [ ] Number of clicks on the sidebar as a negative metric?
+    - [ ] Number of searches now vs before?
+    - [ ] Choice to use dark mode?
```

Comment:
I'd add something around number of discoveries, number of feature flags created etc

Reply:
The growth team has a bunch of PostHog dashboards:
- Activation: https://app.posthog.com/dashboard/60728
- Acquisition: https://app.posthog.com/dashboard/56927
- Revenue: https://app.posthog.com/dashboard/56168

We could reuse these and break the insights down by the feature flag true / false, to see differences between the two variants.

Reply:
My only request: we just need to allow for a bit of burn-in time on any metrics we're evaluating.

We definitely shouldn't be too reactionary to a 5% decline in any given metric in the first week. By default, people hate change, so our first instinct shouldn't just be to revert at the first complaint. It can also take some time for people to get their bearings.

Best analogy I can give: When a car company redesigns a model, the reactions can be polarizing; some people love, a lot of people hate. But does it really affect the adoption of that vehicle in the future? Not necessarily. And over time, people get used to it and move on to complain about the next shiny thing.

PostHog 2999 (as I like to call it – but also known as Base 3000) is a positive change in every regard, but it can take some users time to adjust, even with no substantial differences.

This is less tangible than a minor change in a metric, but if we do have differences in the numbers, I'm hoping we can work to find _better_ solutions to those specific problems than just reverting as a gut reaction.

Reply:
Definitely. I think the way we approach new users is different for how we approach existing users. I think most NEW users won't give us any time to adjust to a new interface. If they hate it (read: they can't do what they want because it's confusing, etc) then they will just leave. At the same time they're less likely to be reactionary based on the prior interface because they've never seen it. I think if we can get to a point where we feel confident about 3000 with new users, existing users will eventually come around.